### DISCUSSION TITLE
GIL in Spyder?

### DISCUSSION THREAD

**Discussion by jgaladepablo** [de-Boer-Lab/GIL#1] [Q&A]
Hello! Thank you so much for developing these tools.
I am trying to make GIL work in Spyder but I am having little success. Could you give more examples of how to use it?
Thank you!

**Comment by nmateyko** [comment]
Hi @jgaladepablo, do you have some examples of how you tried running GIL or any errors that you're getting? I'm not familiar with Spyder but happy to help!

Note that we recently pushed some changes to GIL, so it might be worth trying to reinstall GIL (or pull the changes if you cloned the GIL repo) to see if that fixes any problems. We intend to release an official "v1.0.0" version soon with any future changes made clear with each new release.

**Comment by jgaladepablo** [comment]
Hi @nmateyko,

I figured it out, I think I was running it from the wrong folder and having issues with the GIT.[COMMAND] notation, but as soon as I figured this out I could run it successfully. Thank you!
